syntax = "proto3";

package aws_connect;
import "google/protobuf/timestamp.proto";
message InstanceIdle {
  google.protobuf.Timestamp timestamp = 1;
  string instance_id = 2;
  google.protobuf.Timestamp last_connected_timestamp = 3;
}